tests/server_test.go

```diff
@@ -6289,17 +6289,17 @@ func TestServer_Query_Where_Fields(t *testing.T) {
 		},
 	}...)
 
-	for i, query := range test.queries {
+	var once sync.Once
+	for _, query := range test.queries {
 		t.Run(query.name, func(t *testing.T) {
-			if i == 0 {
+			once.Do(func() {
```

Comment:
ty. These `i == 0` checks drive me nuts. 💯

Reply:
It caused me significant pain when some tests were failing but only if I ran that specific test (since it was still index 3 or whatever, and 0 never ran!)